From 88f0c290084b62887d84d8d507a0cb7a3f3fc1e0 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Wed, 17 Feb 2021 14:25:45 +0000
Subject: [PATCH 092/191] .chj-home/init: make sure the chjize key is imported

This can't be done on the `/etc/skel` files (or at least not easily).
---
 .chj-home/init | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.chj-home/init b/.chj-home/init
index 99d7584..41084ac 100755
--- a/.chj-home/init
+++ b/.chj-home/init
@@ -20,6 +20,8 @@ cancel() {
     exit 1
 }
 
+gpg --import /opt/chj/chjize/cj-key.asc || true
+
 if read -e -p "Please enter your full name: " fullname; then
     if read -e -p "Please enter your (bare) email address: " email; then
 	echo "$fullname" > .chj-home_fullname
-- 
2.47.3

